service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CoreCaseDataStoreLoader.java

```diff
@@ -33,6 +33,10 @@ private CoreCaseDataStoreLoader() {
         new ObjectMapper();        // NO-OP
     }
 
+    public static CaseConverter getCaseConverterInstance() {
```

Comment:
This can removed if it is decided that you don't need to pass case details to all those methods

Reply:
True. Let me know what you think.

Reply:
I still need it as one of the methods will use CaseDetails.